### PR TITLE
Errormessage does now include tagname

### PR DESCRIPTION
**Pull request by Nerzal**
**Describe the PR**
Improved error message, when a markdown file is missing. It does now include the tag name.

### CODE REVIEW

File: parser.go [hunk 1]
```diff
@@ -423,7 +423,7 @@ func getMarkdownFileForTag(tagName string, dirPath string) (string, error) {
 		}
 	}
 
-	return "", errors.New("Unable to find Markdown file in the given directory")
+	return "", errors.New("Unable to find Markdown file for tag '" + tagName + "' in the given directory")
```
Comment (easonlin404) [inline parser.go]: use `fmt.Errorf(format string, a ...interface{})` more suitable if needs formats string.

Comment (Nerzal) [reply parser.go]: ok

**Comment by ubogdan** [comment]
@Nerzal it's a while sicen you started this PR , so in order to fix this I'm going to another PR for this.

**Comment by Nerzal** [comment]
thx @ubogdan

**Pull request closed without merging**